Front-End/src/PageM.js

```diff
@@ -142,7 +142,7 @@ function BumpersDataTable(props) {
 
                 <p className="selection-label" id="Bumpers-Data-Table">Bumpers Data Tables</p>
                 <div className="textArea">
-                    <table border="1" width="100%">
+                    <table border="1" width="200%">
```

Comment:
With this set to `200%`, it ends up putting the table way off the side of the screen. Is this what you wanted, or did you want to fit it all on one screen? 

![image](https://user-images.githubusercontent.com/6099416/222062409-b0d26c11-25b5-4a19-b73d-6412f6eafe57.png)

Reply:
going to fix with future css